protocol/block.re

```diff
@@ -78,6 +78,8 @@ let (hash, verify) = {
         ~handles_hash,
         ~validators_hash,
       );
+    // double hash because tezos always uses blake2b on CHECK_SIGNATURE
```

Comment:
This comment isn't super clear. It would be better to flesh it out and include the link you include in the PR desc.

Reply:
done boss